coins/monero/src/wallet/send/mod.rs

```diff
@@ -624,7 +627,7 @@ impl SignableTransaction {
       // regarding it's view key
       payment = if !modified_change_ecdh {
         if let InternalPayment::Change(change, amount) = &payment {
-          InternalPayment::Payment((change.address, *amount))
+          InternalPayment::Payment((change.address.unwrap(), *amount))
```

Comment:
Prior comment asked about this, here unwraps.

We could also make InternalPayment::Change not internally use the Change struct, avoiding the Option entirely and offering compile-time safety.